.circleci/config.yml

```diff
@@ -782,10 +782,10 @@ workflows:
           chain-id: 11155111
           provider-url: https://sepolia.infura.io/v3/$INFURA_API_KEY
 
-      - fork-test:
+      - fork-test-new:
```

Comment:
And we change this back once it is deployed, correct?

Reply:
yes. Want to get confirmation that new deploy actually works. And when we deploy on chain - flip over to "update"